packages/components/src/Overlay/Popover.test.tsx

```diff
@@ -13,6 +13,13 @@ import { setup } from '../test.utils';
 const theme: Theme = {
   name: 'test',
   components: {
+    Popover: cva(['mt-0.5'], {
```

Comment:
Hm, I think we always need to consider the position, right? The placement has to be passed to the styles inside the component.

Reply:
@sebald  
then should i create a component and create test for it  ?

Reply:
@sebald 
I don't understand

Reply:
I also don't understand @sebald

Reply:
I thought this issue is about to make a space prop where you can put in any value you want 🤔

Reply:
Yes, but adding matching to the top will not work when the popover is shown above the anchor (e g. when there is not enough space before the anchor)